app/modules/create-user/CreateUserScreen.tsx

```diff
@@ -1,7 +1,124 @@
-import { Text } from 'react-native';
+import { FormikProps, useFormik } from 'formik';
+import { Camera } from 'phosphor-react-native';
+import React, { useState } from 'react';
+import {
+  Image,
+  KeyboardAvoidingView,
+  ScrollView,
+  Text,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+import { CustomInput } from '../../components';
 import { Strings } from '../../constants';
-import { FC } from 'react';
+import { useInitializeRefs } from '../../hooks';
+import { Colors, globalMetrics, moderateScale, verticalScale } from '../../theme';
+import { SignUpSchemaTypes, handleSubmitEdit, signUpSchema } from '../../utils';
+import styles from './CreateUserScreenStyles';
+import ProfileOptionsModal from './PictureOptionsModal';
 
-const CreateUserScreen:FC = () => <Text>{Strings.bottomNavigationTitles.create}</Text>;
+const CreateUserScreen = (): JSX.Element => {
+  const [modalShown, setModalShown] = useState<boolean>(false);
+  const formRefs = useInitializeRefs(4);
 
-export default CreateUserScreen;
\ No newline at end of file
+  const handleProfileSelect = (): void => {
+    setModalShown(true);
+  };
+  const formik: FormikProps<SignUpSchemaTypes> = useFormik<SignUpSchemaTypes>({
+    initialValues: {
+      email: '',
+      password: '',
+      firstName: '',
+      lastName: '',
+    },
+    validationSchema: signUpSchema,
+    onSubmit: (values: SignUpSchemaTypes) => {
+      // to be completed in part two
+    },
+  });
+
+  return (
+    <View style={styles.mainContainer}>
+      <View style={styles.top}>
+        <View style={styles.profileImgContainer}>
+          <Image
+            source={{uri: Strings.defaultImg}}
+            style={styles.profileImage}
+            resizeMode="contain"
+          />
+          <TouchableOpacity
+            style={styles.editIcon}
+            onPress={handleProfileSelect}>
+            <Camera size={moderateScale(25)} color={Colors.white} />
+          </TouchableOpacity>
+        </View>
+      </View>
+      <KeyboardAvoidingView
+        style={styles.bottomContainer}
+        behavior={globalMetrics.isIos ? 'padding' : 'height'}
+        keyboardVerticalOffset={verticalScale(15)}>
+        <ScrollView showsVerticalScrollIndicator={false}>
+          <CustomInput
+            editable={true}
```

Comment:
its not required to pass true, for editable here, just mention editable

Reply:
resolved